src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -217,6 +217,13 @@ func runClientProxy(cfg *config.Config) {
 	settings.Configure(cfg, version, buildDate)
 	proxiedsites.Configure(cfg.ProxiedSites)
 
+	httpClient, er := util.HTTPClient(cfg.CloudConfigCA, cfg.Addr)
```

Comment:
Is the CloudConfigCA guaranteed to be good for analytics?  I guess we shouldn't rely on Google using the same CA as S3.

BTW, this reminds me we'll probably need to change the CloudConfigCA to Cloudflare's when we swap in the config server.  Or we could get a cert for the same CA?